quartz/components/Graph.tsx

```diff
@@ -18,6 +18,7 @@ export interface D3Config {
   removeTags: string[]
   showTags: boolean
   focusOnHover?: boolean
+  obsidianLikeFocusOnHover?: boolean
```

Comment:
I think we should just make this behaviour to focusOnHover instead of having this additional feature.

Reply:
Removed in 2f08ab7f643bf3a65073a6aee21292bf6c533dde.